src/vaadin-grid-data-provider-mixin.html

```diff
@@ -159,13 +184,19 @@
       ];
     }
 
+    /** @private */
     _sizeChanged(size) {
       const delta = size - this._cache.size;
       this._cache.size += delta;
       this._cache.effectiveSize += delta;
       this._effectiveSize = this._cache.effectiveSize;
     }
 
+    /**
+     * @param {number} index
+     * @param {HTMLElement} el
+     * @return {void}
```

Comment:
`@return {void}` seems unnecessary and this should probably be `@protected`
```suggestion
     * @protected
```

Reply:
Done.